mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -252,4 +254,15 @@ def test_degenerate():
                       bad_epochs_fname)
     assert_equal(len(w), 1)
 
+
+@requires_version('scipy', '0.12')
```

Comment:
0.12 is our current minimum req so this can be removed alongside any others in the file